deploy/helm/internal-charts/indexer/values.yaml

```diff
@@ -45,12 +45,28 @@ autoReIndex:
 # This config field is available since v2.2.5
 # Previous versions' obsolete are still supported for backward compatible reason
 appConfig:
-  http: 
+  http:
     port: 6103
   elasticSearch:
     serverUrl: http://opensearch:9200
     shards: 1
     replicas: 0
+    indices:
+      datasets:
+        hybridSearch:
+          # -- Configuration for hybrid vector search
+          vectorConfig:
+            mode: "on_disk"
```

Comment:
Let's set to on_memory by default for now (till opensearch supports radial search or allow filter low score result easier)
Other default config might need to adjust according to this.
We probably can add instruction (or typical config) for on_disk to helm doc content (by adding comments) see helm docs here: https://github.com/norwoodj/helm-docs